CGPs/cgp-0127.md

```diff
@@ -0,0 +1,35 @@
+---
+cgp: 127
+title: Update maxDelegateesCount to Improve Governance Flexibility
+date-created: 2024-03-12
+author: "Pavel Hornak (@pahor167)"
+status: DRAFT
+discussions-to: [Insert forum discussion link]
+governance-proposal-id: [To be assigned]
+date-executed: TBD
+---
+
+## Overview
+
+This proposal seeks to update the `maxDelegateesCount` in the Celo governance framework. By increasing the limit on the number of delegatees, we're aiming to enhance the flexibility of vote delegation. This change is designed to accommodate the varying needs of Celo holders, recognizing that some may prefer to entrust their voting power to multiple delegates who can actively participate in the governance process on their behalf. This adjustment is a step towards optimizing governance participation, ensuring that it can be as effective and inclusive as possible.
```

Comment:
```suggestion
This proposal seeks to update the `maxDelegateesCount` in the Celo governance promess. By increasing the limit on the number of delegatees, we're aiming to enhance the flexibility of vote delegation. This change is designed to accommodate the varying needs of Celo holders, recognizing that some may prefer to entrust their voting power to multiple delegates who can actively participate in the governance process on their behalf. This adjustment is a step towards optimizing governance participation, ensuring that it can be as effective and inclusive as possible.
```